src/task/MonitorTask.ts

```diff
@@ -249,6 +259,20 @@ export default class MonitorTask
       }
     );
 
+    for (const policy of this.videoEncodingHealthPolicies) {
+      this.applyHealthPolicy(policy, connectionHealthData, () => {
+        this.degradeVideoCodec(policy.name);
+        switch (policy.name) {
+          case 'Video Encoding CPU Health':
```

Comment:
This feels a bit off. Could we try casting? Or maybe we can add a unit test on the name with a comment mentioning that there is a dependency?

Reply:
Yeah. I added a enum for video encoding health policy names and added comments there to note the names are used in monitor task to determine degradation reason and emit metrics